Trustroots/trustroots
# Copy: use localized format for date

#### Proposed Changes

* Change to localized date format so that translators don't need to change this according to their own locale manually.

From https://momentjs.com/docs/#/displaying/format/

Month name, day of month, year | LL | September 4, 1986
-- | -- | --
  | ll | Sep 4, 1986



#### Testing Instructions

* Add contact, then remove/revoke them:
    <img width="311" alt="Screenshot 2021-03-21 at 16 29 54" src="https://user-images.githubusercontent.com/87168/111909124-a4223500-8a64-11eb-91c6-92bc0781c738.png">
    <img width="688" alt="Screenshot 2021-03-21 at 16 30 05" src="https://user-images.githubusercontent.com/87168/111909085-8523a300-8a64-11eb-8d2b-614a9d88a7e9.png">
* Open any profile and see in the sidebar when they joined
    
    <img width="278" alt="Screenshot 2021-03-21 at 16 29 11" src="https://user-images.githubusercontent.com/87168/111909120-9ff61780-8a64-11eb-83ab-b187cbcc47d9.png">

* See "launched" text at http://localhost:3000/media
* See "Enabling the latent trust" text at http://localhost:3000/statistics
